src/test/java/seedu/address/logic/parser/FindCommandParserTest.java

```diff
@@ -38,7 +37,20 @@ public void parse_validSingleNameArg_returnsFindCommand() throws ParseException
     }
 
     @Test
-    public void parse_validSingleCourseArg_returnsFindCommand() throws ParseException {
+    public void parse_validSpacedNameArg_returnsFindCommand() {
+        FindCommand expectedFindCommand = new FindCommand(
+                new NameContainsKeywordsPredicate(List.of("Alice Tan"))
+        );
+
+        // no leading and trailing whitespaces
+        assertParseSuccess(parser, " n/Alice Tan", expectedFindCommand);
+
+        // multiple whitespaces around keyword
+        assertParseSuccess(parser, " \n n/Alice Tan\n  \t", expectedFindCommand);
```

Comment:
I don't think that User is able to input indentations or new line in the name, so the inclusion of "\n" as a testcase isn't entirely necessary? But it's nice that you've considered such cases.